flask_monitoringdashboard/frontend/sass/custom.css

```diff
@@ -507,4 +507,10 @@ footer.sticky-footer {
     color: red;
     vertical-align: super;
     font-size: small;
+}
+
+@media (min-width: 1500px) {
+    .container {
+        max-width: 1600px !important;
```

Comment:
shouldn't both these be the same?
or at least the max-width smaller than min-width?

Reply:
Honestly I have no idea. ```@media (min-width: 1200px){.container{max-width:1140px}}``` looks like you are right though. I think I understood it backwards.
I changed the min-width to 1644px and it still looks good for me.